src/CrawlTestLvl/private/groupobjbox.cpp

```diff
@@ -0,0 +1,54 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the GPLv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#include "boxitem.h"
+#include "groupobjbox.h"
+
+namespace TestLvl {
+
+GroupObjBox::GroupObjBox(): CRAWL::IWorldItem("GroupObjBox") {
+
+    setDistance(20);
+    changeLayout(CRAWL::Refresh::CIRCLE);
+
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+    add(new BoxItem);
+
+}
+
+void GroupObjBox::render(unsigned int tbfMsec)
+{
+    Layout::render(tbfMsec);
```

Comment:
you forget invoke the render function of the world item parent 
```suggestion
    Layout::render(tbfMsec);
    IWorldItem::render(tbfMsec);
```